src/renderer_webgpu.cpp

```diff
@@ -8,15 +8,14 @@
 //#define DAWN_ENABLE_BACKEND_D3D12
 #define DAWN_ENABLE_BACKEND_VULKAN
 
+#define BGFX_CONFIG_DEBUG_ANNOTATION 0
+
 #if BGFX_CONFIG_RENDERER_WEBGPU
 #	include "renderer_webgpu.h"
+#	include "renderer_vk.h"
```

Comment:
Why do you need this?!

Reply:
For now shaderc is still storing the samplers/buffers types using their VK type id.
Also, when the backend is Vulkan we need to initialize the swapchain (which is why it was in the ifdef below, but because of the uniform types from shaderc, had to be moved outside, until we change this)

Reply:
Ok then just do:

```
#define VK_NO_STDINT_H
#define VK_NO_PROTOTYPES
#include <vulkan-local/vulkan.h>
```